src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball ball = new Ball();
```

Comment:
There's no need to create an instance of Ball here since it's immediately replaced in the loop on line 8. You can directly use the instance returned from `lottery.getRandomBall()`.